kybra/compiler/dfx_extension/extension.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "kybra",
+    "version": "0.0.0",
```

Comment:
How should versioning work? If this matches the version of kybra then we'll never have confusion about which version is which...we should get this to update automatically then in our publish script or something